Makefile

```diff
@@ -1,6 +1,8 @@
 COMMONENVVAR=GOOS=linux GOARCH=amd64
 BUILDENVVAR=CGO_ENABLED=0
+TOPOLOGYAPI_MANIFESTS=https://raw.githubusercontent.com/k8stopologyawareschedwg/noderesourcetopology-api/master/manifests
 
+KUBECTL ?= kubectl
```

Comment:
Maybe better to call it `CLI` or `KUBECLI` instead of `KUBECTL` here?

Reply:
used `KUBECLI`